diff --git a/node_modules/webpack/types.d.ts b/node_modules/webpack/types.d.ts
index b064287..b63e7d2 100644
--- a/node_modules/webpack/types.d.ts
+++ b/node_modules/webpack/types.d.ts
@@ -11030,10 +11030,6 @@ declare abstract class SortableSet<T> extends Set<T> {
 	getFromUnorderedCache<R>(fn: (arg0: SortableSet<T>) => R): R;
 	toJSON(): T[];
 
-	/**
-	 * Iterates over values in the set.
-	 */
-	[Symbol.iterator](): IterableIterator<T>;
 }
 declare class Source {
 	constructor();
